src/rust/engine/process_execution/build.rs

```diff
@@ -0,0 +1,4 @@
+// Copyright 2025 Pants project contributors (see CONTRIBUTORS.md).
+// Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+fn main() {}
```

Comment:
Why is an empty `build.rs` needed?

Reply:
I no longer recall, I think it had to do with building protos, but deleting it seems to still work, so...